typings/codePush.d.ts

```diff
@@ -1,400 +0,0 @@
-// Type definitions for Apache Cordova CodePush plugin.
```

Comment:
the following removed files are cordova plugin d.ts files that i accidentally checked in